### ISSUE TITLE
"No checklist found for tag" error shown for pre-existing tag.

### ISSUE THREAD

**Issue by jluna79** [delashum/obsidian-checklist-plugin#148]
Hi,

I've run into a bug that I cannot escape from and is breaking my workflow dramatically. I used to have an "act" tag that I would use for different tasks (i.e. "act/todo", "act/review", etc).

Since last month checklists have stopped working and a "No checklist found for tag #act" is always shown.

Here's the configuration:

<img width="751" alt="image" src="https://user-images.githubusercontent.com/411374/184342238-665351aa-faef-4386-bf40-f8e20a41ac1e.png">

And here's what I keep seeing:

<img width="1005" alt="image" src="https://user-images.githubusercontent.com/411374/184342330-1631b24e-7fe6-42c1-8d8d-a5e7b0cb9417.png">

Another weird thing I see: the settings menu does not show the correct information for the configured tag.
**Edit**: Also (as mentioned in a comment below) I cannot click on the tags to show/hide anything

<img width="327" alt="image" src="https://user-images.githubusercontent.com/411374/184342758-813ee880-07eb-4d45-ad86-75f8a894d0a3.png">

**Versions**
- Checklist plugin: `2.2.7`
- Obsidian: `0.15.9` (macOS)

Thanks!

**Comment by JellEllis** [comment]
I am having similar problems.

As well as the sometimes malformed tag names in the Show Tags? dialogue, I am sometimes finding that I can't click on any of the tag names shown to select the ones I need.


Checklist version 2.2.5
Obsidian v0.15.9 (Windows 11)

**Comment by flankstaek** [comment]
I can't replicate this issue on my machines. Would you mind clearing out your tags and re-inputting them fresh?

If you're still seeing the bug can you open up the Developer Console and screenshot if there are any errors?

**Comment by jluna79** [comment]
Using the Tag Wrangler plugin I renamed `act` to `actt` and now items are showing :| (weird). 

However, new items are not added to the list. 

![image](https://user-images.githubusercontent.com/411374/186201612-3fb7a04f-9e40-4112-9052-210e61c5aa7e.png)

After re-renaming tags back to `act` things seem to go back to normal:
![image](https://user-images.githubusercontent.com/411374/186202068-b20ddef3-4c50-4ec8-bfcd-0b6d24fcc8d5.png)

**But** as soon as I close the Vault and re-open it... 😢 

![image](https://user-images.githubusercontent.com/411374/186202305-db90b3d9-34fe-4ca1-a482-2fad2316e3b3.png)

**Comment by wuzhen97** [comment]
I am having similar problems.

![image](https://user-images.githubusercontent.com/61906420/188050750-03a88bc3-b4c2-4b96-b66c-0460e47b68b3.png)

**Maybe it's because of the multi-level tags.**

**Comment by jluna79** [comment]
Any update on this @flankstaek ? Did the console screenshots help?

**Comment by dammitcoetzee** [comment]
also have this, can no longer see any todo items checklist 2.2.10, Obsidian v1.0.3

**Comment by heavenideas** [comment]
Same here v2.2.10 of the plugin and v1.0.3 of obsidian

**Comment by DennisVargas** [comment]
The plugin had been working great up until today when I was no longer able to add any more todo items and this error arose. 

This occurred on two of my machines, at first I still had the todo checklists visible up until I unchecked the todo tag as a filter and they all disappeared. The filter option was then was no longer selectable until the Obsidian program was fully reset. Observing the developer console I am noticing a similar error message as above. 

![image](https://user-images.githubusercontent.com/28330008/211407515-2f2cfe5a-d800-45fa-a0cc-0ba6fad25bba.png)

The last few days of using the plugin have been great so I am thrilled with the workflow possibilities using this plugin. 
I'm rather inspired to fork the code and begin searching for a solution since it does not seem to be a bug that everyone is experiencing.

**Comment by jluna79** [comment]
I don't know if this plugin is still maintained. Last time I heard from @flankstaek was 5 months ago and didn't get an answer to the updated info I added :(.

This plugin was super useful in my workflow while I used it. If you _do_ fork the code (or become a collaborator) would be super helpful!

**Comment by jluna79** [comment]
Still no answer nor fix :(

Came here to share I'm now using a Dataview query to simulate this plug-in (not optimal but it works).

1. Install [Obsidian Dataview](https://github.com/blacksmithgu/obsidian-dataview).
1. Add the same tag to the files where your tasks will be written (I use `#my/daily-notes`)
2. Use the tag you would usually list on the `obsidian-checklist` panel as a header for sets of tasks

Example
````
### #todo
- [ ] some todo
- [ ] another
````

4. Create a new note with the following Dataview query

```
```dataview  
Task  
From #my/daily-notes 
Where !completed and (text != "" and text != "...")
Sort file.ctime desc
Limit 20  
Group by meta(section).subpath

```

Hope this helps other people having this problem.

**Comment by delashum** [comment]
Fixed in 2.2.13

**Comment by jluna79** [comment]
Finally, it works again! Thanks!
